src/WaxSigningApi.ts

```diff
@@ -208,15 +237,18 @@ export class WaxSigningApi {
         verified,
         signatures,
         whitelistedContracts,
-        serializedTransaction
+        serializedTransaction,
+        startTime
       } = event.data;
 
       if (!verified || !signatures) {
         throw new Error("User declined to sign the transaction");
       }
 
       this.whitelistedContracts = whitelistedContracts || [];
-
+      if(startTime && startTime > 0) {
+        this.metricLog("login.metric.manual_sign_transaction_time", Math.floor((new Date).getTime()) - startTime, []);
```

Comment:
this can get changed to `this.metricLog(metricName, Math.floor((new Date).getTime()) - startTime, []);`

Reply:
this repeated code `Math.floor((new Date).getTime())` should probably be changed into a helper currentTime()

Reply:
why is there the word "login" on the front of this metric by the way? login.metric.manual_sign_transaction_time. I think "waxjs.siging.manual" would be way more clear, and then you could use "waxjs.siging.endpoint" for the endpoint timeing metric